### ISSUE TITLE
It's Cleaning Time! 🧹

### ISSUE THREAD

**Issue by MIlhamDedi** [MIlhamDedi/bbmerlion#1]
# Messy code, me no like
Yes, the project was bootstrapped from a template, and is messy as hell. We need some cleaning.

## Current Condition
Current used views and components are hard edit from the template. You might find commented out *"unnecessary"* code in many used files.

## Expected Condition
Recode component files that are *"hotfix"* to the clean ones. You **SHOULD** leave unused components for future use. You **MAY** change component level routings.

## tl;dr
Bikin komponen2 baru yang ga ada komen2an
